serenata_toolbox/chamber_of_deputies/dataset.py

```diff
@@ -7,18 +7,19 @@
 import pandas as pd
 from .reimbursements import Reimbursements
 
+
 class Dataset:
 
-    YEARS = [n for n in range(2009, date.today().year+1)]
+    AVAILABLE_YEARS = [year for year in range(2009, date.today().year + 1)]
 
-    def __init__(self, path):
+    def __init__(self, path, years=AVAILABLE_YEARS):
         self.path = path
-
+        self.years = years
```

Comment:
Wouldn't it be nice to have this argument flexible? Year could be an `int` (eg. `Dataset('/tmp`, years=2010)`) or a `list` (eg. `Dataset('/tmp`, years=[2010, 2011])`). In that case I'd replace this line by:

```python
        self.years = [years] if isinstance(year, int) else years
```